From 3be6b6703aa0a9431b788d08c4d26d6209be3316 Mon Sep 17 00:00:00 2001
Message-ID: <3be6b6703aa0a9431b788d08c4d26d6209be3316.1731337861.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Mon, 11 Nov 2024 16:08:56 +0100
Subject: [PATCH] setup.py: do not bundle c and c++ libs in wheel

We want to package libs separately, so do not include them in the wheel
(and, consequently, not in the python package).
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index f740931c8..1330da7b5 100755
--- a/setup.py
+++ b/setup.py
@@ -334,8 +334,6 @@ def build():
                 ]
     else:
         names = [
-                pipcl.get_soname(f'{build_dir()}/libmupdf.so'),     # C.
-                pipcl.get_soname(f'{build_dir()}/libmupdfcpp.so'),  # C++.
                 f'{build_dir()}/_mupdf.so',                         # Python internals.
                 f'{build_dir()}/mupdf.py',                          # Python.
                 ]
-- 
2.47.0.346.g8c6d622db9

